python/requirements/ml/requirements_dl.txt

```diff
@@ -1,20 +1,15 @@
 # These requirements are used for the CI and CPU-only Docker images so we install CPU only versions of torch.
-# For GPU Docker images, you should install requirements_ml_docker.txt instead.
+# For GPU Docker images, you should install requirements_ml_docker.txt afterwards.
 
+tensorflow~=2.6.2
+tensorflow-probability~=0.14.1
```

Comment:
Is keras installed as a dependency in tensorflow automatically?

Reply:
exactly. There was a small problem back in the day (#20032) for a single version, but that's fixed and so tensorflow will request the right version of keras